packages/react-components/react-dialog/src/utils/localShorthands.ts

```diff
@@ -0,0 +1,8 @@
+export function gridArea(name: string) {
```

Comment:
FYI: it's implemented in [Griffel](https://github.com/microsoft/griffel/blob/main/packages/core/src/shorthands/gridArea.ts) and should be available after #24114.